config/config.go

```diff
@@ -50,6 +50,17 @@ const (
 	EnableProfilerKey = "ENABLE_PROFILER"
 	// StatsIntervalKey defines interval for printing basic tdex statistics
 	StatsIntervalKey = "STATS_INTERVAL"
+	// ExperimentalKey defines wether to use an Elements node as the daemon's
+	// block explorer instead of the default Esplora service.
+	ExperimentalKey = "EXPERIMENTAL"
+	// ElementsRPCHostKey is the RPC host of the Elements node. Experimental ONLY.
+	ElementsRPCHostKey = "ELEMENTS_RPC_HOST"
```

Comment:
I would do different here
```suggestion
	ElementsJsonRpcEndpointKey = "ELEMENTS_JSON_RPC_ENDPOINT"
```

Winch would be in the canonical URL form of `protocol://user:pass@host:port`

The golang url package provides `url.Parse` to handle this